cranelift/codegen/src/isa/x64/abi.rs

```diff
@@ -92,7 +92,11 @@ fn in_vec_reg(ty: types::Type) -> bool {
     }
 }
 
-fn get_intreg_for_arg_systemv(idx: usize) -> Option<Reg> {
+fn get_intreg_for_arg_systemv(call_conv: &CallConv, idx: usize) -> Option<Reg> {
+    assert!(match call_conv {
+        CallConv::SystemV | CallConv::BaldrdashSystemV => true,
+        _ => false,
```

Comment:
If it's an outright bug that should never happen, then +1 for panicing rather than continuing (returning `false` or whatever).  I'm not convinced that's the case here, though (can't infer)

Reply:
`CallConv::Fast` and `CallConv::Cold` can also reach this I think.